core/src/main/java/org/arquillian/smart/testing/configuration/Configuration.java

```diff
@@ -122,6 +134,7 @@ public void setAutocorrect(boolean autocorrect) {
         configItems.add(new ConfigurationItem("disable", SMART_TESTING_DISABLE, false));
         configItems.add(new ConfigurationItem("debug", SMART_TESTING_DEBUG, false));
         configItems.add(new ConfigurationItem("autocorrect", SMART_TESTING_AUTOCORRECT, false));
+        configItems.add(new ConfigurationItem("customStrategies", Pattern.compile(SMART_TESTING_CUSTOM_STRATEGIES_PATTERN)));
```

Comment:
When you removed the pattern from the `ConfigurationItem` then the second parameter doesn't make sense. There is no such a support for getting system properties based on the pattern or provided prefix in the `ObjectMapper` class - that should be implemented there as part of this PR, otherwise, you don't get the values set by system properties there.